pkgs/by-name/tg/tg-archive/package.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "tg-archive";
+  version = "1.1.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "knadh";
+    repo = pname;
```

```suggestion
    repo = "tg-archive";
```

`repo = pname;` is good for DRY but creates a binding that goes too far, please take a look at: https://github.com/nix-community/nixpkgs-lint/issues/21